_docs/_latest/configure/notifier/index.md

```diff
@@ -176,37 +176,78 @@ notifier:
+
+#### Setup
+1. Select `Add Security Sources` on the Cloud SCC Beta Dashboard.
+
+1. Find the [Forseti Cloud SCC Connector](https://console.cloud.google.com/marketplace/details/forseti/forseti-security-cloud-scc-connector)
+in Cloud Marketplace.
+
+1. Follow the step-by-step on-boarding flow triggered from the Forseti card.
+- Choose the project that is hosting Forseti
+- Use the existing Forseti server service account (which will be assigned the
+`Security Center Findings Editor` role)
+- Note: The on-boarding flow will generate a source_id and assign the `Security
+  Center Findings Editor` role, which is required to write to the Cloud SCC Beta API
+  to surface the findings in the Cloud SCC.
+
+1. Enable the Cloud SCC Beta API in the Forseti project either via either
+the UI or API: 
+- via the UI (`API & Services ->  Library`)
+- via the command line in Cloud Shell `gcloud services enable securitycenter.googleapis.com`
+- Note: You will need to have either owner, editor or service management roles
+  in the Project in order to enable the API
+
+1. At the org-level, go to `IAM & Admin -> IAM` and verify that the
+`Security Center Findings Editor` role has been added to your Forseti server
+service account 
+
+1. Enable the  API connector config to Cloud SCC.  Specifically this means,
+in the Forseti project server bucket, edit the `configs/forseti_conf_server.yaml`,
+ as follows:
+
+- Open `forseti-security/configs/forseti_conf_server.yaml`.
+- Navigate to the `notifier` > `violation` > `cscc` section.
+
+  The options below are available for you to configure:
 
 * `enabled:`
   * **Description**: Whether to send notification to Cloud SCC.
-  * **Valid values**: one of valid `true` or `false`
+  * **Valid values**: one of valid **`true`** or `false`
 
 * `mode:`
   * **Description**: How to send the violations to Cloud SCC.
-  * **Valid values**: one of valid `api` or `bucket`
-  * **Note**: `api` mode will only work if Forseti is [using a service account from the Cloud Security Center project](https://cloud.google.com/security-command-center/docs/how-to-programmatic-access).
+  * **Valid values**: one of valid **`api`** or `bucket`
+  * **Note**: `api` mode will only work if Forseti is [using a service account
+  from the Cloud Security Center project](https://cloud.google.com/security-command-center/docs/how-to-programmatic-access).
 
 * `organization_id:`
   * **Description**: The organization id.
   * **Valid values**: String
   * **Note**: Must be in the form of `organizations/12345`. Used only in `api` mode.
 
-* `gcs_path`
-  * **Description**: The path to a Cloud Storage bucket.
+* `source_id`
+  * **Description**: ID from the Cloud SCC beta on-boarding. **This must be added**
+  to use the Beta API integration.
   * **Valid values**: String
-  * **Note**: Must start with `gs://`. Used only in `bucket` mode.
+  * **Note**: It’s in the form: source_id: <organizations/ORG_ID/sources/SOURCE_ID>
```

Comment:
Minor but maybe extend the contraction so `it's` becomes `it is` to be more in line with the grammar in the doc, else take out entirely and just go with `In the form:`.

Additionally: Should put backticks around <organizations/ORG_ID/sources/SOURCE_ID> instead of just having it as plain text.